GitCommands/Git/GitModule.cs

```diff
@@ -1364,7 +1364,8 @@ public string FormatPatch(string from, string to, string output, int? start = nu
                 {
                     "-M -C -B",
                     { start != null, $"--start-number {start}" },
-                    $"{from.Quote()}..{to.Quote()}",
+                    { !string.IsNullOrEmpty(from), $"{from.Quote()}..{to.Quote()}" },
+                    { string.IsNullOrEmpty(from), $"--root {to.Quote()}" },
```

Comment:
```suggestion
                    { !string.IsNullOrEmpty(from), $"{from.Quote()}..{to.Quote()}", $"--root {to.Quote()}" },
```

I prefer the normal outcome first here, others may prefer a non negated test

Reply:
I can just make an if outside the initializer, to make it clearer. Thoughts?

Reply:
Fixed as requested for now.